### ISSUE TITLE
IllegalAccessError on player connect

### ISSUE THREAD

**Issue by SlimeDog** [InsightsPlugin/InsightsWorldEditExtension#7]
Spigot 1.17.1-3226
Insights 6.7.0

On player connect
```
[10:14:37] [Insights-worker-7/WARN]: java.lang.IllegalAccessError: class dev.frankheijden.insights.api.concurrent.containers.ChunkContainer tried to access field net.minecraft.world.level.entity.PersistentEntitySectionManager.f (dev.frankheijden.insights.api.concurrent.containers.ChunkContainer is in unnamed module of loader org.bukkit.plugin.java.PluginClassLoader @10007044; net.minecraft.world.level.entity.PersistentEntitySectionManager is in unnamed module of loader 'app')
[10:14:37] [Insights-worker-7/WARN]:    at dev.frankheijden.insights.api.concurrent.containers.ChunkContainer.get(ChunkContainer.java:134)
[10:14:37] [Insights-worker-7/WARN]:    at dev.frankheijden.insights.api.concurrent.containers.ChunkContainer.get(ChunkContainer.java:24)
[10:14:37] [Insights-worker-7/WARN]:    at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1764)
[10:14:37] [Insights-worker-7/WARN]:    at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
[10:14:37] [Insights-worker-7/WARN]:    at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
[10:14:37] [Insights-worker-7/WARN]:    at java.base/java.lang.Thread.run(Thread.java:831)
```

**Comment by SlimeDog** [comment]
Multiples of this WARN occur when player changes world, or executes (some) commands, etc.

```
[11:04:08] [Insights-worker-3/WARN]: java.lang.IllegalAccessError: class dev.frankheijden.insights.api.concurrent.containers.ChunkContainer tried to access field net.minecraft.world.level.entity.PersistentEntitySectionManager.f (dev.frankheijden.insights.api.concurrent.containers.ChunkContainer is in unnamed module of loader org.bukkit.plugin.java.PluginClassLoader @15c8eb25; net.minecraft.world.level.entity.PersistentEntitySectionManager is in unnamed module of loader 'app')
[11:04:08] [Insights-worker-3/WARN]:    at dev.frankheijden.insights.api.concurrent.containers.ChunkContainer.get(ChunkContainer.java:134)
[11:04:08] [Insights-worker-3/WARN]:    at dev.frankheijden.insights.api.concurrent.containers.ChunkContainer.get(ChunkContainer.java:24)
[11:04:08] [Insights-worker-3/WARN]:    at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1764)
[11:04:08] [Insights-worker-3/WARN]:    at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
[11:04:08] [Insights-worker-3/WARN]:    at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
[11:04:08] [Insights-worker-3/WARN]:    at java.base/java.lang.Thread.run(Thread.java:831)
```

**Comment by FrankHeijden** [comment]
Fixed in [6.7.1](https://www.spigotmc.org/resources/insights-super-configurable-region-limits-asynchronous-scans-1-17-x.56489/update?update=420385)

**Comment by SlimeDog** [comment]
Confirmed fix in 6.7.1. Thanks.
